external-crates/move/crates/move-compiler/src/cfgir/translate.rs

```diff
@@ -652,12 +673,36 @@ fn function_body(
                     &UniqueMap::new(),
                     &mut cfg,
                 );
+                if context
+                    .env
+                    .supports_feature(context.current_package, FeatureGate::Move2024Optimizations)
```

Comment:
Can we remove this feature yet? Do we want to?

Reply:
We have other things gated behind it today. I'd rather remove that gate for everything in a follow-up PR.